### PR TITLE
FIX: GCS airspeed sensor selection

### PR DESCRIPTION
**Pull request by BrainFPV**
Fixes the airspeed sensor type selection, which had been completely broken.

### CODE REVIEW

**Comment by mlyle** [comment]
Awesome!  I've wondered why this was.

I know nothing about the specific pitot sensors out there.  Is it safe to assume if someone checks that box they want the eagletree sensor and not the diydrones options we support?

File: ground/gcs/src/plugins/config/configmodulewidget.cpp [hunk 1]
```diff
@@ -620,21 +618,46 @@ void ConfigModuleWidget::updateAirspeedGroupbox(UAVObject *obj)
...
+ */
+void ConfigModuleWidget::enableAirspeedTypeGPS(bool checked)
+{
+    if (checked){
+        AirspeedSettings *airspeedSettings;
+        airspeedSettings = AirspeedSettings::GetInstance(getObjectManager());
+        AirspeedSettings::DataFields airspeedSettingsData;
+        airspeedSettingsData = airspeedSettings->getData();
+        airspeedSettingsData.AirspeedSensorType = AirspeedSettings::AIRSPEEDSENSORTYPE_GPSONLY;
+        airspeedSettings->setData(airspeedSettingsData);
```
Comment (guilhermito) [inline ground/gcs/src/plugins/config/configmodulewidget.cpp]: Aren't you entering in an endless loop here, this is called whenever the airspeedsettings are updated and you are updating them inside this.

Comment (BrainFPV) [inline ground/gcs/src/plugins/config/configmodulewidget.cpp]: Fortunately this isn't the case; the `clicked` signal only gets emitted when the user clicks the checkbox, not when it is changed using `setChecked`.

**Comment by BrainFPV** [comment]
@mlyle the default Pitot selection is arbitrary but was biased by the fact that I have one of the ET sensors ;). We could change the default to the `MPXV7002` sensor, which can be bought from [3DR](https://store.3drobotics.com/products/airspeed-kit-with-mpxv7002dp).

**Comment by mlyle** [comment]
Whatever you think is best.  I'm OK with this to merge.

**Pull request merged**